website/src/views/timetable/ModulesTableFooter.tsx

```diff
@@ -30,12 +30,16 @@ type Props = {
   semester: Semester;
   moduleTableOrder: ModuleTableOrder;
   modules: Module[];
+  hiddenInTimetable: string[];
 
   setModuleTableOrder: (moduleTableOrder: ModuleTableOrder) => void;
 };
 
 const ModulesTableFooter: React.FC<Props> = (props) => {
   const totalMCs = sumBy(props.modules, (module) => parseFloat(module.moduleCredit));
+  const hiddenMCs = props.modules
```

Comment:
This is MCs of modules _not_ hidden 

```suggestion
  const shownMCs = props.modules
```